Ambermoon.net/Configuration.cs

```diff
@@ -191,7 +189,7 @@ public static string ExecutableDirectoryPath
 
                 var assemblyDirectory = Path.GetDirectoryName(assemblyPath);
 
-                if (isWindows)
+                if (OperatingSystem.IsWindows())
```

Comment:
Is this working on Mac now? Do you use Visual Studio for Mac? I guess you could add ` || OperatingSystem.IsMacOS()` to allow game data inside the project directory.